### PR TITLE
[Feat]:Grid component added

### PR DESCRIPTION
**Pull request by t007rushi**
GRID :
-> 2 items in grid
-> 3 items in grid

### CODE REVIEW

File: components/grid.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.grid-two {
+  grid-template-columns: 1fr 1fr;
+}
+.grid-three {
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.bgGrey {
+  background-color: var(--overlay-bg-color);
+}
```
Comment (horsemaker) [inline components/grid.css]: Could we make it `bg-grey`, so that it's more self-explanatory?

File: components/grid.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+.bgred {
+  background-color: var(--btn-primary-color);
+}
+.bggreen {
+  background-color: var(--border-two);
+}
+.bgblue {
+  background-color: var(--white-color);
+}
```
Comment (horsemaker) [inline components/grid.css]: These classes too, `bg-red`, `bg-green` & `bg-blue`?

File: components/grid.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+@media only screen and (max-width: 625px) {
+  .grid-two {
+    grid-template-columns: 1fr;
+  }
+  .grid-three {
+    grid-template-columns: 1fr 1fr ;
+  }
+}
```
Comment (horsemaker) [inline components/grid.css]: Is this the correct way to make them responsive? I think no media query is needed for the grids here.

**Review by horsemaker — APPROVED** [review]

**Review by bossmonkey98 — APPROVED** [review]

File: components/grid.css (hunk 2 above)
Comment (t007rushi) [reply components/grid.css]: right thank you for this one

**Pull request merged**